tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```diff
@@ -725,7 +725,8 @@ class ReplicationUtility {
             }
             const { srcData, destMetadata, destData } = data;
             assert.strictEqual(srcData.ReplicationStatus, 'COMPLETED');
-            assert.strictEqual(srcData.ContentLength, destMetadata[0].size);
+            assert.strictEqual(`${srcData.ContentLength}`,
```

Comment:
Why are we changing this?

Reply:
I believe it was because the SDK changed the output from number to string